src/brush.js

```diff
@@ -227,7 +227,7 @@ function brush(dim) {
             function tween(t) {
               state.selection = t === 1 && selection1 === null ? null : i(t);
               redraw.call(that);
-              emit.brush();
+              emit.brush(event);
```

Comment:
It doesn’t feel appropriate to use the _event_ here because time has passed—we’re now inside a transition.